Makefile

```diff
@@ -83,7 +83,7 @@ ENVTEST_K8S_VERSION = 1.24.2
 
 # The target deployment environment, that corresponds to the Kustomize directory
 # used to build the manifests.
-ENV ?= default
+ENV ?= manifests
```

Comment:
Could you please make sure `make deploy` takes the image into account correctly with `cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}`. It may be the tests pass because the default image is used, while it should be using the locally built one.

Reply:
It seems there's a conflict.
While we have the image set to `$(odh-codeflare-operator-controller-image)`  in `manager.yaml` Kustomize cannot overwrite it with the passed image because you can't use the variable in the command e.g. 
`$(KUSTOMIZE) edit set image $(odh-codeflare-operator-controller-image)=${IMG}`.

We could revert that variable back to `controller:latest` and alter the make file so that if there was no image passed it would default to the latest image in params.env. wdyt?

Reply:
Would replacing `$(KUSTOMIZE) edit set image $(odh-codeflare-operator-controller-image)=${IMG}` with something that changes the value in `params.env` with the value of `IMG` be option?

Reply:
Absolutely we can use a similar perl command from the workflow portion. I'll get on that. Thanks!